src/test/java/com/codejoust/main/mapper/GameMapperTests.java

```diff
@@ -112,11 +124,19 @@ public void toDto() {
 
         assertEquals(RoomMapper.toDto(room), gameDto.getRoom());
         assertEquals(1, gameDto.getPlayers().size());
-        assertEquals(ProblemMapper.toDto(problem), gameDto.getProblems().get(0));
         assertEquals(game.getPlayAgain(), gameDto.getPlayAgain());
         assertEquals(game.getAllSolved(), gameDto.getAllSolved());
         assertEquals(game.getGameEnded(), gameDto.getGameEnded());
 
+        assertEquals(TestFields.PROBLEM_NAME, gameDto.getProblems().get(0).getName());
+        assertEquals(TestFields.PROBLEM_DESCRIPTION, gameDto.getProblems().get(0).getDescription());
+        assertEquals(ProblemDifficulty.MEDIUM, gameDto.getProblems().get(0).getDifficulty());
+
+        assertEquals(TestFields.INPUT, gameDto.getProblems().get(0).getTestCases().get(0).getInput());
+        assertEquals("", gameDto.getProblems().get(0).getTestCases().get(0).getOutput());
+        assertEquals("", gameDto.getProblems().get(0).getTestCases().get(1).getInput());
+        assertEquals("", gameDto.getProblems().get(0).getTestCases().get(1).getOutput());
```

Comment:
nit: Save `gameDto.getProblems().get(0)` and the associated `.getTestCases()` method in a variable so the same get methods don't have to be repeated multiple times.